app/models/async_request/job.rb

```diff
@@ -1,6 +1,41 @@
 module AsyncRequest
   class Job < ActiveRecord::Base
     serialize :params, Array
-    enum status: [:waiting, :processing, :processed]
+    enum status: { waiting: 0, processing: 1, processed: 2, failed: 3 }
+
+    def self.find_by_jwt_token(jwt_job_id)
```

Comment:
Why you need this? in jobs_controller you don't use it

Reply:
I forgot to remove that